diff --git a/InstallFullDB.sh b/InstallFullDB.sh
index f765b17c..8ae43186 100755
--- a/InstallFullDB.sh
+++ b/InstallFullDB.sh
@@ -2782,6 +2782,21 @@ function auto_script_install_world()
   true
 }
 
+# apply core updates using config file settings and normal user
+function auto_script_apply_core_update()
+{
+  show_mysql_settings
+  echo
+  echo "Applying all the latest core updates..."
+
+  if ! apply_core_update; then
+    false
+    return
+  fi
+
+  true
+}
+
 # do a backup
 function auto_script_backup()
 {
@@ -2844,6 +2859,9 @@ function show_help
   echo "   -World"
   echo "    Install world db only using none root user defined in $CONFIG_FILE"
   echo
+  echo "   -UpdateCore"
+  echo "    Apply core updates using none root user defined in $CONFIG_FILE"
+  echo
   echo "   -InstallAll rootuser rootpass"
   echo "    Install all db by droping previous ones and recreate them from scratch"
   echo "    Require root access with arg1 as root username and arg2 as root password"
@@ -2908,6 +2926,14 @@ if [[ "$1" = "-World" ]]; then
 
   exit 0
 fi
+# only apply core updates using config
+if [[ "$1" = "-UpdateCore" ]]; then
+  if ! auto_script_apply_core_update; then
+    exit 1
+  fi
+
+  exit 0
+fi
 
 # only show config
 if [[ "$1" = "-Config" ]]; then
@@ -2935,4 +2961,4 @@ if [[ "$1" != "" ]]; then
 fi
 
 # launch main menu
-main_menu
\ No newline at end of file
+main_menu
